From: <hesso@pool.math.tu-berlin.de>
Subject: Comment out the paragraph for glib-1
 It is no longer needed and AM_PATH_GLIB is not available unless we
 also depend on the old glib.

--- gdome2-0.8.1+debian.orig/configure.in
+++ gdome2-0.8.1+debian/configure.in
@@ -63,18 +63,18 @@
 	GLIB_1=yes
 )
 
-if test "x$GLIB_1" = "xyes"
-then
-	PKG_CHECK_MODULES(GLIB, glib)
-	GLIB_REQUIRED="glib"
-	GLIB_MIN_VERSION=1.2.10
-        AM_PATH_GLIB($GLIB_MIN_VERSION,,AC_MSG_ERROR(Could not find GLIB (see config.log for details).))
-else
+#if test "x$GLIB_1" = "xyes"
+#then
+#	PKG_CHECK_MODULES(GLIB, glib)
+#	GLIB_REQUIRED="glib"
+#	GLIB_MIN_VERSION=1.2.10
+#        AM_PATH_GLIB($GLIB_MIN_VERSION,,AC_MSG_ERROR(Could not find GLIB (see config.log for details).))
+#else
 	PKG_CHECK_MODULES(GLIB, glib-2.0)
 	GLIB_REQUIRED="glib-2.0"
 	GLIB_MIN_VERSION=2.2.0
         AM_PATH_GLIB_2_0($GLIB_MIN_VERSION,,AC_MSG_ERROR(Could not find GLIB (see config.log for details).))
-fi
+#fi
 AC_SUBST(GLIB_MIN_VERSION)
 AC_SUBST(GLIB_LIBS)
 AC_SUBST(GLIB_REQUIRED)
